.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
```

Comment:
- Instead of using the custom logic to handle commit details, consider using a more structured approach like parsing the commit messages and descriptions using a dedicated library or tool.